mon-pix/app/components/user-certifications-detail-header.js

```diff
@@ -18,6 +18,10 @@ export default class UserCertificationsDetailHeader extends Component {
     return parseISODateOnly(this.args.certification.birthdate);
   }
 
+  get isFrenchDomainExtension() {
+    return this.currentDomain.isFranceDomain;
+  }
+
```

Comment:
Question : 
Je ne suis pas trop au courant des décisions PIX sur la gestion des langues. Mais je trouve dommage qu'ici, nous soyons en .org mais en français et que le lien n'apparaisse pas. N'y a-t-il pas moyen de connaitre la langue utilisé plutôt que de baser l'affichage du lien en fonction de l'extension de domaine ?

Reply:
Je pense en effet qu'il s'agit d'une erreur d'interprétation du besoin de ma part.
Je me renseigne :)

Reply:
Je pense que @yaf a raison, le .org peut etre utilise pour passer des certifs FR.
Je propose de se baser sur la langue configurée par l'utilisateur venant du `/me`, je crois accessible via `this.currentUser.user.lang`